ppr-ui/public/config/configuration.json

```diff
@@ -24,5 +24,8 @@
   "ADDRESS_COMPLETE_KEY": "",
   "SYSTEM_MESSAGE": "<a style='text-decoration: underline; color:inherit;' href='https://www2.gov.bc.ca/gov/content/employment-business/business/managing-a-business/permits-licences/news-updates/modernization/personal-property-registry-transition' target='_blank'>See known issues and updates for Personal Property Registry here</a>.",
   "SYSTEM_MESSAGE_TYPE": "warning",
-  "SITEMINDER_LOGOUT_URL": "https://logontest7.gov.bc.ca/clp-cgi/logoff.cgi"
+  "SITEMINDER_LOGOUT_URL": "https://logontest7.gov.bc.ca/clp-cgi/logoff.cgi",
+  "REGISTRIES_SEARCH_API_URL": "https://bcregistry-dev.apigee.net/registry-search",
+  "REGISTRIES_SEARCH_API_VERSION": "/api/v1",
+  "REGISTRY_SEARCH_API_KEY": "oPiLjOE4rvmw556dTextKI7mHqfSd7AF"
```

Comment:
Will need to request that these configurations are added to our 1password i believe. 
Might need to discuss with Doug or Thor or whomever manages Assets configurations.

Reply:
Also this an item that should be called out in a future release ticket as a release dependency for each environment.

Reply:
ok I'll get in touch with Doug.